gmso/utils/files/spce.xml

```diff
@@ -0,0 +1,22 @@
+<ForceField version="0.0.1" name="SPC/E Water" combining_rule="geometric">
```

Comment:
I think this file already exists in the `utils/files/gmso_xmls/test_ff_styles` directory. I think you can probably delete it and source from there.